packages/core/content-releases/admin/src/pages/ReleaseDetailsPage.tsx

```diff
@@ -11,108 +11,109 @@ import {
   Main,
   Popover,
   Typography,
+  Tr,
+  Td,
 } from '@strapi/design-system';
-import { CheckPermissions, useAPIErrorHandler, useNotification } from '@strapi/helper-plugin';
+import {
+  AnErrorOccurred,
+  CheckPermissions,
+  LoadingIndicatorPage,
+  PageSizeURLQuery,
+  PaginationURLQuery,
+  Table,
+  useAPIErrorHandler,
+  useNotification,
+  useQueryParams,
+} from '@strapi/helper-plugin';
 import { ArrowLeft, EmptyDocuments, More, Pencil, Trash } from '@strapi/icons';
 import { useIntl } from 'react-intl';
 import { useParams } from 'react-router-dom';
 import styled from 'styled-components';
 
+import { GetReleaseActions } from '../../../shared/contracts/release-actions';
 import { ReleaseModal, FormValues } from '../components/ReleaseModal';
 import { PERMISSIONS } from '../constants';
 import { isAxiosError } from '../services/axios';
-import { useUpdateReleaseMutation } from '../services/release';
+import {
+  GetReleaseActionsQueryParams,
+  useUpdateReleaseMutation,
+  useGetReleaseQuery,
+  useGetReleaseActionsQuery,
+} from '../services/release';
+/* -------------------------------------------------------------------------------------------------
+ * ReleaseDetailsLayout
+ * -----------------------------------------------------------------------------------------------*/
+const ReleaseInfoWrapper = styled(Flex)`
+  align-self: stretch;
+  border-bottom-right-radius: ${({ theme }) => theme.borderRadius};
+  border-bottom-left-radius: ${({ theme }) => theme.borderRadius};
+  border-top: 1px solid ${({ theme }) => theme.colors.neutral150};
+`;
 
 const PopoverButton = styled(Flex)`
   align-self: stretch;
 `;
 
 const PencilIcon = styled(Pencil)`
-  width: 16px;
-  height: 16px;
+  width: ${({ theme }) => theme.spaces[4]};
+  height: ${({ theme }) => theme.spaces[4]};
   path {
     fill: ${({ theme }) => theme.colors.neutral600};
   }
 `;
 
 const TrashIcon = styled(Trash)`
-  width: 16px;
-  height: 16px;
+  width: ${({ theme }) => theme.spaces[4]};
+  height: ${({ theme }) => theme.spaces[4]};
   path {
     fill: ${({ theme }) => theme.colors.danger600};
   }
 `;
 
-const ReleaseInfoWrapper = styled(Flex)`
-  align-self: stretch;
-  border-radius: 0 0 4px 4px;
-  border-top: 1px solid ${({ theme }) => theme.colors.neutral150};
-`;
+const countDays = (date: string) => {
+  const currentDateInMillis = new Date().getTime();
+  const startDateInMilliseconds = new Date(date).getTime();
+  const timeDiff = currentDateInMillis - startDateInMilliseconds;
 
-const ReleaseDetailsPage = () => {
+  const daysPassed = Math.floor(timeDiff / (1000 * 60 * 60 * 24));
+
+  return daysPassed;
+};
```

Comment:
You can use [formatRelativeTime](https://formatjs.io/docs/react-intl/api/#formatrelativetime) from react-intl for this. [Here](https://github.com/strapi/strapi/blob/develop/packages/core/admin/admin/src/content-manager/pages/EditView/components/Information.tsx#L58) is an example of how to use it from the content manager

Reply:
I saw your example, yes I can do it, my solution shows just the day passed not the minutes or hours